code/src/index.css

```diff
@@ -5,9 +5,59 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+.message-container {
+  width: 700px;
+  margin: auto;
+  margin-top:20px;  
+  margin-bottom: 20px; 
+  padding:20px; 
+  font-size:16px;
+  box-shadow: 10px 5px black;
+  border: 1px solid;
+  background-color: rgb(228, 228, 228);
+
+}
+
+.thoughts-container {
```

Comment:
would be nice to add some kind of rule here that cuts really long words at the end of the container (for instance if someone adds a reeeeeally long word it now floats over the container border). Check out the attribute overflow for instance :)